benchmarks/sysbench/Portfile

```
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 1.0
+PortGroup github 1.0
+
+github.setup            akopytov sysbench 1.0.20
+revision                1
+categories              benchmarks
+platforms               darwin
+maintainers             @sainnhe \
+                        gmail.com:sainnhe
+license                 GPL-2+
+
+description             Scriptable database and system performance benchmark
+
+long_description \
+    sysbench is a scriptable multi-threaded benchmark tool based on LuaJIT. It is \
+    most frequently used for database benchmarks, but can also be used to create \
+    arbitrarily complex workloads that do not involve a database server.
+
+homepage                https://github.com/akopytov/sysbench
+
+checksums               rmd160  ee4e41c3a5f5f18c80787bc608f84faf33f1c989 \
+                        sha256  bab986be195206acdaa6609c7cf4556d2e2e222806cca44569683a6e04e00ac9 \
+                        size    1509764
+
+depends_build-append    port:pkgconfig \
+                        port:automake
+
+depends_lib             port:libtool \
+                        path:lib/libssl.dylib:openssl
+
+depends_run             port:luajit
```

libtool is only needed at build time, not runtime. sysbench links with the luajit library so it is needed at build and runtime (i.e. `depends_lib`). It also linked with the umem library on my system, so let's add that dependency too.
```suggestion
depends_lib             port:luajit \
                        path:lib/libssl.dylib:openssl \
                        port:umem
```